src/routes/SEO.svelte

```diff
@@ -0,0 +1,38 @@
+<script lang="ts">
+	import { seo } from '$lib/stores/SeoStore';
+
+	export let title = 'listd';
+	export let description = 'listd';
+
+	seo.set({
+		title,
+		description,
+	});
```

Comment:
When this component mounts, it resets the store (server-side store vs client-side store, i think), re"set" it in `onMount` call.
```ts
function loadSeo() {
	seo.set({
		title,
		description,
	})
}

// First time
loadSeo()

// Second time
onMount(() => {
  loadSeo()
})
```

Reply:
Good call, updated.